_source/logzio_collections/_prometheus-sources/telegraf-mongo.md

```diff
@@ -76,3 +85,70 @@ The full list of data scraping and configuring options can be found [here](https
 {% include metric-shipping/generic-dashboard.html %} 
 
 </div>
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="atlas">
+
+## Overview
+
+Telegraf is a plug-in driven server agent for collecting and sending metrics and events from databases, systems and IoT sensors.
+
+To send your Prometheus-format Mongo Atlas metrics to Logz.io, you need to add the **inputs.mongodb** and **outputs.http** plug-ins to your Telegraf configuration file.
+
+#### Configuring Telegraf to send your metrics data to Logz.io
+
+<div class="tasklist">
+
+##### Set up Telegraf v1.17 or higher
+
+{% include metric-shipping/telegraf-setup.md %}
+
+##### Add the inputs.mongodb plug-in
+
+First you need to configure the input plug-in to enable Telegraf to scrape the Mongo Atlas data from your hosts. To do this, add the following code to the configuration file:
+
+
+``` ini
+[[inputs.mongodb]]
+  servers = ["mongodb://<<USER-NAME>>:<<PASSWORD>>@<<SERVER-NAME>>:27017"]
```

Comment:
its better to put example with several servers as mongo atlas is minimum of 3 servers cluster